include/rppdefs.h

```diff
@@ -22,28 +29,28 @@ typedef unsigned long long  Rpp64u;
 typedef long long           Rpp64s;
 typedef float               Rpp32f;
 typedef double              Rpp64f;
-typedef void*              RppPtr_t;
+typedef void*               RppPtr_t;
+typedef size_t              RppSize_t;
 
 typedef enum
 {
-    RPP_SUCCESS = 0,
-    RPP_ERROR   = 1,
+    RPP_SUCCESS             = 0,
+    RPP_ERROR               = 1,
 } RppStatus;
 
- typedef enum
- {
-    rppStatusSuccess = 0,
-    rppStatusBadParm   = 1,
-    rppStatusUnknownError = 2,
+typedef enum
+{
+    rppStatusSuccess        = 0,
+    rppStatusBadParm        = 1,
```

Comment:
it is better to give -ve values for error codes

Reply:
Done